compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -427,11 +427,22 @@ LLVMRustInlineAsm(LLVMTypeRef Ty, char *AsmString, size_t AsmStringLen,
                   char *Constraints, size_t ConstraintsLen,
                   LLVMBool HasSideEffects, LLVMBool IsAlignStack,
                   LLVMRustAsmDialect Dialect, LLVMBool CanThrow) {
+#if LLVM_VERSION_GE(13, 0)
   return wrap(InlineAsm::get(unwrap<FunctionType>(Ty),
                              StringRef(AsmString, AsmStringLen),
                              StringRef(Constraints, ConstraintsLen),
                              HasSideEffects, IsAlignStack,
                              fromRust(Dialect), CanThrow));
+#else
+  if (CanThrow) {
+    report_fatal_error("Shouldn't get called on older versions");
```

Comment:
Is this reachable when trying to use unwinding inline asm on older LLVM versions? If so this should be a proper rustc error.

Reply:
I agree.